### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (119)
Screenshot (120)

Please review the landing page and homepage of my note-taking app.

### CODE REVIEW

**Review by Jia-A — APPROVED** [review]
- Good UI.
- Nice Folder structure.
- Good use of CSS variables.
- Good naming conventions.
You can merge it.

File: src/note-components/Landingpage/landingpage.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.main-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```
Comment (dev-enforced) [inline src/note-components/Landingpage/landingpage.css:5]: Can use the min-height attribute too in place of height!! Would help in height adjustment while viewing the same section on phone

File: src/note-components/Sidebar/sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <p>Archieve</p>
+    </div>
+    <div className="list left-icon">
+        <span><i class="far fa-trash text-color"></i></span>
+        <p>Trash</p>
+    </div>
+    <div className="list left-icon">
+        <span><i class="far fa-user-circle text-color"></i></span>
+        <p>Profile</p>
+    </div>
```
Comment (dev-enforced) [inline src/note-components/Sidebar/sidebar.jsx:23]: Can have a list of nav links which you can map over
```js
Can have a list of colors which you can map over

[n1,n2,n3,n4,n5],map((everyLink,index)=>{
return (
<div key={everyLink} className=`list left-icon`>
<p>{everyLink}</p>
</div>
)
})
```

**Review by dev-enforced — APPROVED** [review]
The UI looks really awesome. Looking forward to see the live preview of Note Cave
Would advise the use of prettier formatter to help you format the files and helps in better readability!!Gave some minor suggestions which you can add on later but do look at them to help remove repetitive code blocks

File: src/note-components/Landingpage/landingpage.css (hunk 1 above)
Comment (Megha-Joshi) [reply src/note-components/Landingpage/landingpage.css:5]: Okay!

File: src/note-components/Sidebar/sidebar.jsx (hunk 2 above)
Comment (Megha-Joshi) [reply src/note-components/Sidebar/sidebar.jsx:23]: Okay

**Pull request merged**